lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -138,16 +126,12 @@
     });
   };
 
-  // Makes a reasonable determination that
-  // the data passed to the function is formatted
-  // JSON and returns `true`, if so. False, if not
-
-  function isJSON (json, callback) {
-    if (!json) {
-      return callback(new Error('You forgot to pass JSON data as the first argument.'));
+  function maybeJson (json, callback) {
+    if (typeof json !== 'string') {
+      return callback(new Error('You forgot to pass a string as the first argument.'));
```

Comment:
Just return false. No callback invocation that makes the flow of the code confusing, and no need for an error object.

Reply:
Unfortunately this didn't make this morning's release, but if it gets wrapped up soon it could make the next one...